core/src/display_object/edit_text.rs

```diff
@@ -831,6 +931,118 @@ impl<'gc> EditText<'gc> {
                 .firing_variable_binding = false;
         }
     }
+
+    pub fn get_selection(self) -> Option<TextSelection> {
+        self.0.read().selection
+    }
```

Comment:
Nit: Rust naming convention should be `selection` here (I know we're not very consistent with this yet).